R/data-objects.R

```diff
@@ -223,31 +240,50 @@ local_to_irods_ <- function(
 #'
 #' @return The R object in case of `ireadRDS()`, invisibly `NULL` in case of
 #'  `iget()`.
+#' @seealso
+#'  [iput()] for sending files,
+#'  [isaveRDS] for sending R objects to iRODS,
```

Comment:
here and next line: add parentheses (`[isaveRDS()]` and `[readRDS()]`)

Reply:
done